src/components/Triangle/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+
+const Triangle = ({ row, col, values, appConfig }) => {
+
+  const { side, height, styleTarsiaTriangle, triangleYTextPadding, styleTarsiaText } = appConfig;
```

Comment:
Destructured the props object that's passed to the component, as well as the subsequent `appConfig` object: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring